python/orca/src/bigdl/orca/common.py

```diff
@@ -308,24 +316,26 @@ def init_orca_context(cluster_mode=None, runtime="spark", cores=2, memory="2g",
                 sc = init_spark_standalone(num_executors=num_nodes, executor_cores=cores,
                                            executor_memory=memory, **spark_args)
             else:
-                raise ValueError("cluster_mode can only be local, yarn-client, yarn-cluster,"
-                                 "k8s-client or standalone, "
-                                 "but got: %s".format(cluster_mode))
-        ray_args = {}
-        for key in ["redis_port", "password", "object_store_memory", "verbose", "env",
-                    "extra_params", "num_ray_nodes", "ray_node_cpu_cores", "include_webui",
-                    "system_config"]:
-            if key in kwargs:
-                ray_args[key] = kwargs[key]
-        from bigdl.orca.ray import OrcaRayContext
-        ray_ctx = OrcaRayContext(runtime="spark", cores=cores, num_nodes=num_nodes,
-                                 sc=sc, **ray_args)
+                invalidInputError(False,
+                                  "cluster_mode can only be local, yarn-client, yarn-cluster,"
+                                  "k8s-client or standalone, "
+                                  "but got: %s".format(cluster_mode))
+            ray_args = {}
+            for key in ["redis_port", "password", "object_store_memory", "verbose", "env",
+                        "extra_params", "num_ray_nodes", "ray_node_cpu_cores", "include_webui",
+                        "system_config"]:
+                if key in kwargs:
+                    ray_args[key] = kwargs[key]
+            from bigdl.orca.ray import OrcaRayContext
+            ray_ctx = OrcaRayContext(runtime="spark", cores=cores, num_nodes=num_nodes,
+                                     sc=sc, **ray_args)
```

Comment:
Shouldn't remove the indent for these code blocks...
See this PR: https://github.com/intel-analytics/BigDL/pull/4638
@cyita will help revert this in a new PR